examples/common/python/config/config.py

```diff
@@ -104,3 +109,25 @@ def parse_configuration_file(filename, variable_map) :
         text = Template(text).substitute(variable_map)
 
     return toml.loads(text)
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+
+def read_config_from_toml(input_file, config_name = None, confpaths = [".", TCFHOME + "/" + "config"]):
```

Comment:
remove space around = for function arguments with default values

Reply:
Done.